Makefile

```diff
@@ -110,6 +110,16 @@ install-helm:
 
 .PHONY: e2e-local-bootstrap
 e2e-local-bootstrap:
-	kind create cluster --image kindest/node:v${KIND_K8S_VERSION}
+	kind create cluster --image kindest/node:v1.18.2@sha256:7b27a6d0f2517ff88ba444025beae41491b016bc6af573ba467b70c5e8e0d85f
 	make image
-	kind load docker-image --name kind $(DOCKER_IMAGE):$(IMAGE_VERSION)
+	kind load --name kind docker-image $(DOCKER_IMAGE):$(IMAGE_VERSION)
+	# Create Dev namespace for local e2e-testing
+	kubectl create ns dev
+ifdef DEVCONTAINER
+	# curl secrets store CSI Driver into dev container.
+	git clone https://github.com/bdlb77/secrets-store-csi-driver.git  /go/src/secrets-store-csi-driver
```

Comment:
Now that the helm charts are packaged for the driver, we could get rid of this dependency on the driver repo and just using `helm add repo ... && helm install`. WDYT?

Reply:
I removed it altogether. In the Bats testing  the installation of the provider using Helm also installs the CSI Driver too correct?